google/cloud/testing_util/timer.cc

```diff
@@ -22,19 +22,22 @@ namespace testing_util {
 
 #if GOOGLE_CLOUD_CPP_HAVE_GETRUSAGE
 namespace {
-int rusage_who() {
-#if GOOGLE_CLOUD_CPP_HAVE_RUSAGE_THREAD
-  return RUSAGE_THREAD;
-#else
+int rusage_who(CpuAccounting accounting) {
+  switch (accounting) {
+    case CpuAccounting::kPerThread:
+      return RUSAGE_THREAD;
```

Comment:
Does this imply `RUSAGE_THREAD` is available on all `getrusage()` platforms (i.e., no need for `GOOGLE_CLOUD_CPP_HAVE_RUSAGE_THREAD`)?

Reply:
Nope, I just forgot about macOS.  Will send a fix in a jiffy.